src/activity_handlers.h

```diff
@@ -29,7 +29,7 @@ enum butcher_type : int {
     DISSECT         // dissect a corpse for CBMs
 };
 
-int butcher_time_to_cut( const player &u, const item &corpse_item, butcher_type action );
+int butcher_time_to_cut( const player &u, const item &corpse_item, const butcher_type action );
```

Comment:
This will be reverted by the clang-tidy-plugin.

In function declarations, the top-level const qualifiers have no meaning. `void foo( const int )` and `void foo( int )` declare the exact same function and are from the point of view of the compiler equivalent.